src/openapi.yaml

```diff
@@ -18,6 +18,24 @@ paths:
 
   # TODO: add paths here
 
+  # /users endpoints
+  /users/{user_id}/entries:
+    $ref: "./paths/users/user_id/entries.yaml"
+
+  /users/{user_id}/entries/{entry_id}:
+    $ref: "./paths/users/user_id/entries/entry_id/entry.yaml"
+
+  /users/{user_id}/reviews:
```

Comment:
When do you think this request is being called?

I currently do not see a scenario where a user might request all reviews of a specific user. As far as I know this is confidential information therefor a user should only be able to find reviews in the context of an entry, they are associated with.

Please correct me if I am missing something.